contracts/src/interfaces/IHyperdriveDeployerCoordinator.sol

```diff
@@ -59,6 +59,10 @@ interface IHyperdriveDeployerCoordinator {
     ///         duration or is not a multiple of the checkpoint duration.
     error InvalidPositionDuration();
 
+    /// @notice Thrown when the sender of a `deploy`, `deployTarget`, or
+    ///         `initialize` transaction isn't the associated factory.
+    error InvalidSender();
```

Comment:
If this is only used in the `onlyFactory` modifier, then should we make the name more specific?  The comment describes a much, much more specific use for the error than the name of the error implies.